ex7.5_NumPy__vector_matrix.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def create_vector_to_matrix(matrix: np, vector: np):
+    """
+    function that add a vector to each row of a given matrix.
+    :param matrix: given matrix
+    :param vector: given vector
+    :return:
+    """
+    for row in range(matrix.shape[0]):
+        matrix[row] += vector
```

Comment:
The beauty of numpy is that in most cases for loops are not needed (and are usually less efficient), try using broadcasting.
https://numpy.org/doc/stable/user/basics.broadcasting.html